src/uu/fold/src/fold.rs

```diff
@@ -220,11 +220,14 @@ fn fold_file<T: Read>(mut file: BufReader<T>, spaces: bool, width: usize) {
 
             match ch {
                 '\t' => {
-                    if col_count + 8 > width && !output.is_empty() {
+                    let next_tab_stop = col_count + 8 - col_count % 8;
```

Comment:
Could you please either move 8 into a variable or document why 8?
Thanks

Reply:
Done! :)